common/reviews/api/load-themed-styles.api.ts

```diff
@@ -0,0 +1,60 @@
+// @public
+enum ClearStyleOptions {
+  // (undocumented)
+  all = 3,
+  // (undocumented)
+  onlyNonThemable = 2,
+  // (undocumented)
+  onlyThemable = 1
+}
+
+// @public
+export function clearStyles(option: ClearStyleOptions = ClearStyleOptions.all): void;
+
+// @public
+export function configureLoadStyles(loadStylesFn: ((processedStyles: string, rawStyles?: string | ThemableArray) => void) | undefined): void;
+
+// @public
+export function configureRunMode(mode: Mode): void;
+
+// @public
+export function detokenize(styles: string | undefined): string | undefined;
+
+// @public
+export function flush(): void;
```

Comment:
>flush [](start = 16, length = 5)

Pete's magic wand.

Reply:
We're happy with this.

---
In reply to: [147273753](https://github.com/Microsoft/web-build-tools/pull/395#discussion_r147273753) [](ancestors = 147273753)